### DISCUSSION TITLE
Does disconnecting a tcp connection directly using `HttpRequest.io().unwrap().close()` lead to unpredictable behavior

### DISCUSSION THREAD

**Discussion by Jhonfunk** [ntex-rs/ntex#420] [Q&A] [answered]
Does ntex have a reliable way to directly disconnect a tcp connection without replying to it. (Used for some DDOS-like requests)

**Comment by fafhrd91** [comment]
it is not clear what is the problem?

**Reply by fafhrd91** [reply]
it is not clear what is the question

**Reply by Jhonfunk** [reply]
@fafhrd91


Simple question: can I just close the tcp connection without sending a Http response?


Another question: is HttpRequest.io().unwrap().close() safe? I mean will it cause ntex to run improperly.

**Reply by fafhrd91** [reply, marked as answer]
io close will close tcp connection and it is safe to call
